bin/configs/scala-akka-http-server.yaml

```diff
@@ -0,0 +1,6 @@
+generatorName: scala-akka-http-server
+outputDir: samples/server/petstore/scala-akka-http-server
+inputSpec: modules/openapi-generator/src/test/resources/3_0/petstore.yaml
+templateDir: modules/openapi-generator/src/main/resources/scala-akka-http-server
+additionalProperties:
+  artifactId: openapi-scala-akka-http-server
```

Comment:
Thanks for the PR.

There's already one in ./bin/configs/other/scala-akka-http-server.yaml

Please move that one to `bin/configs/` instead.

Reply:
@wing328 

> Please move that one to bin/configs/ instead (or remove that one)

I've deleted `./bin/configs/other/scala-akka-http-server.yaml`
(That file was same as `bin/configs/scala-akka-http-server.yaml`)